hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go

```diff
@@ -531,6 +534,28 @@ func (mh *Memcached) implementingMonitoring() {
 		")",
 		metricsFragment)
 	pkg.CheckError("adding metrics content", err)
+
+	// Create metricsdocs directory
+	err := os.Mkdir(filepath.Join(mh.ctx.Dir, "monitoring/metricsdocs"), os.ModePerm)
+	pkg.CheckError("creating metricsdocs directory", err)
+
+	// Create metricsdocs file
+	_, err = os.Create(filepath.Join(mh.ctx.Dir, "monitoring/metricsdocs/metricsdocs.go"))
+	pkg.CheckError("creating metricsdocs file", err)
+
+	metricsdocsPath := filepath.Join(mh.ctx.Dir, "monitoring/metricsdocs/metricsdocs.go")
```

Comment:
you are already doing this in line 543, do this before and use the var there

Reply:
So fix that in lines 521, 524 as well?

Reply:
yes, you change there too, it's the same path in both of them

Reply:
Done